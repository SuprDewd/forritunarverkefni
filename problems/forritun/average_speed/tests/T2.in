00:03:49
00:05:46
00:12:53 36
00:16:38
00:20:51 113
00:27:43
00:31:47
00:37:16
00:42:39
00:46:02
00:48:34 93
00:51:53 115
00:53:17
01:01:23
01:06:50 69
01:10:22
01:11:05
01:15:52
01:16:55 49
01:19:31 70
01:22:27 77
01:24:24
01:30:11
01:36:19
01:38:57
01:42:02
01:48:45
01:48:48 197
01:50:53
01:56:25 128
01:59:12 188
02:05:55
02:09:36
02:17:08 85
02:18:35 148
02:26:04
02:34:19 175
02:41:25
02:49:30 30
02:51:01
02:53:14 94
02:55:02
02:57:53 173
03:01:30 98
03:03:54 44
03:07:19
03:15:13
03:18:41 173
03:24:54
03:27:59 109
03:35:59 194
03:39:23 169
03:46:13 126
03:49:15 124
03:55:18 167
03:56:00 67
03:57:00 61
04:02:52 62
04:08:02
04:11:47 55
04:17:59 28
04:23:35
04:23:51 192
04:28:08
04:30:24
04:33:03
04:35:27
04:40:21
04:44:35
04:47:26
04:53:15
05:01:18
05:02:00 140
05:04:50 149
05:06:32
05:09:27 174
05:16:40
05:23:39 191
05:25:07
05:30:15
05:30:38
05:37:43 133
05:41:00
05:44:02
05:51:33
05:53:34 130
05:59:42
06:07:54 100
06:08:34
06:16:14
06:16:31
06:22:11 167
06:27:37 127
06:28:47
06:32:00
