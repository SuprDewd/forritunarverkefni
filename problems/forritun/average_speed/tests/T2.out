00:03:49 0.00 km
00:05:46 0.00 km
00:16:38 2.25 km
00:27:43 17.71 km
00:31:47 25.37 km
00:37:16 35.70 km
00:42:39 45.84 km
00:46:02 52.21 km
00:53:17 64.80 km
01:01:23 80.33 km
01:10:22 94.84 km
01:11:05 95.66 km
01:15:52 101.16 km
01:24:24 110.42 km
01:30:11 117.84 km
01:36:19 125.71 km
01:38:57 129.09 km
01:42:02 133.05 km
01:48:45 141.67 km
01:50:53 148.57 km
02:05:55 193.72 km
02:09:36 205.26 km
02:26:04 249.38 km
02:41:25 290.44 km
02:51:01 314.77 km
02:55:02 318.70 km
03:07:19 340.02 km
03:15:13 345.82 km
03:24:54 366.28 km
04:08:02 453.19 km
04:23:35 465.36 km
04:28:08 479.19 km
04:30:24 486.44 km
04:33:03 494.92 km
04:35:27 502.60 km
04:40:21 518.28 km
04:44:35 531.83 km
04:47:26 540.95 km
04:53:15 559.56 km
05:01:18 585.32 km
05:06:32 598.40 km
05:16:40 626.57 km
05:25:07 651.49 km
05:30:15 667.83 km
05:30:38 669.05 km
05:41:00 698.88 km
05:44:02 705.60 km
05:51:33 722.26 km
05:59:42 740.02 km
06:08:34 758.90 km
06:16:14 771.68 km
06:16:31 772.15 km
06:28:47 799.19 km
06:32:00 805.99 km
